features/manage-position/YourPosition.tsx

```diff
@@ -126,7 +131,11 @@ const YourPosition = () => {
               {`${_collateral} ${_collSymbol}`}
             </Status>
             <Status>
-              <Label>Tokens outstanding: </Label>
+              <Label>Collateral backing debt: </Label>
```

Comment:
I thought about adding casework here to show the old text if you don't have a withdrawal pending and show this text if you do. However, I actually think it's cleaner like you've got it here because it becomes very clear exactly what your risk exposure is.

Reply:
yeah might as well show everything